src/Routes/Categories/Categories.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Categories = () => (
+  <div>Categories</div>
```

Comment:
- [ ] Kindly put a button with the text `Check status` as stated in the project requirements. 😎️

**Project Requirements**
![Screenshot from 2023-02-24 10-34-00](https://user-images.githubusercontent.com/17243325/221121306-1b710159-97cc-4e7f-8a5a-4ef2525dd26e.png)

Reply:
Sure, I will add the required text. Thanks for adding this information